lib/src/main.dart

```diff
@@ -231,18 +240,22 @@ class _TextFieldTagsState extends State<TextFieldTags> {
       },
       onChanged: (value) {
         if (_showValidator == false) {
-          if (value.contains(" ") || value.contains(",")) {
-            final List<String> splitedTagsList = value.multiSplit([",", " "]);
+          var containedSeparator = widget.textSeparators
+              .cast<String?>()
+              .firstWhere((element) => value.contains(element!), orElse: () => null);
+          if (containedSeparator == null) return;
+
+            final List<String> splitedTagsList = value.split(containedSeparator);
             final int indexer = splitedTagsList.length > 1
                 ? splitedTagsList.length - 2
                 : splitedTagsList.length - 1;
             final String lastLastTag =
                 splitedTagsList[indexer].trim().toLowerCase();
             if (lastLastTag.length > 0) {
-              _textEditingController!.clear();
+              _textEditingController.clear();
               if (!_tagsStringContents!.contains(lastLastTag)) {
-                if (widget.validator == null ||
-                    widget.validator!(lastLastTag) == null) {
+                final validatorResult = widget.validator!(lastLastTag);
```

Comment:
Reverse this because if user doesn't implement `validator`, it gives of `null` where you can't `null!`.

Reply:
Done! Thank you!